pkg/codegen/codegen.go

```diff
@@ -351,17 +366,30 @@ func (b *Builder) WriteTemplate(ctx context.Context, filePath,
 				cmd := matches[2]
 				isCommand = true
 
+				log := b.log.WithField("command.name", cmd)
+				log.Debug("Processing command")
+
 				switch cmd {
 				case "Block":
 					blockName := matches[3]
 
+					log.WithFields(logrus.Fields{
+						"block.name":  blockName,
+						"block.start": filePath + ":" + strconv.Itoa(i),
```

Comment:
```suggestion
						"block.start": fmt.Sprintf("%s:%s", filePath, strconv.Itoa(i)),
```
string concatenation is ugly

Reply:
ur ugly

Reply:
```suggestion
						"block.start": fmt.Sprintf("%s:%d", filePath, i),
```